frontend/public/components/factory/list.jsx

```diff
@@ -241,7 +240,6 @@ export class Rows extends React.Component {
             {({width}) => <div ref={registerChild}>
               <VirtualList
                 autoHeight
-                data={data}
```

Comment:
I think we need to pass data to tell the list to rerender when data changes, but there may be a better way.